src/wallet/wallet.h

```diff
@@ -831,7 +836,14 @@ class CWallet final : public FillableSigningProvider, private interfaces::Chain:
     uint256 m_last_block_processed GUARDED_BY(cs_wallet);
 
     //! Fetches a key from the keypool
-    bool GetKeyFromPool(CPubKey &key, bool internal = false);
+    bool GetKeyFromPool(CPubKey& key, bool internal = false);
+
+    /* Height of last block processed is used by wallet to know depth of transactions
+     * without relying on Chain interface beyond asynchronous updates. For safety, we
+     * initialize it to -1.
+     */
+    int m_last_block_processed_height{-1};
+    GUARDED_BY(cs_wallet);
```

Comment:
I'm not sure if this macro does anything?

Reply:
Thanks, I misused some linter scripts I think